pkg/asset/manifests/gcp/cluster.go

```diff
@@ -0,0 +1,140 @@
+package gcp
+
+import (
+	"fmt"
+	"net"
+	"os"
+
+	"github.com/apparentlymart/go-cidr/cidr"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	capg "sigs.k8s.io/cluster-api-provider-gcp/api/v1beta1"
+
+	"github.com/openshift/installer/pkg/asset"
+	"github.com/openshift/installer/pkg/asset/installconfig"
+	"github.com/openshift/installer/pkg/asset/manifests/capiutils"
+)
+
+// GenerateClusterAssets generates the manifests for the cluster-api.
+func GenerateClusterAssets(installConfig *installconfig.InstallConfig, clusterID *installconfig.ClusterID) (*capiutils.GenerateClusterAssetsOutput, error) {
+	manifests := []*asset.RuntimeFile{}
+
+	networkName := installConfig.Config.GCP.Network
+	if installConfig.Config.GCP.NetworkProjectID != "" {
+		// Subnets should exist in the network with name NetworkProjectID when provided
+		networkName = installConfig.Config.GCP.NetworkProjectID
+	}
+	if networkName == "" {
+		// no network was provided, this will be the name
+		networkName = fmt.Sprintf("%s-network", clusterID.InfraID)
+	}
+
+	// auto create subnets is not used as the subnet information is provided through
+	// the install config
+	autoCreateSubnets := false
```

Comment:
If this is supposed to always be `false`, let's make it a const: 
```suggestion
	const autoCreateSubnets = false
```